code/handlers/logusage.q

```diff
@@ -34,17 +34,21 @@ nextid:{:id+::1}
 // A handle to the log file
 logh:@[value;`logh;0]
 
-// write a query log message
+// write a query log message, direct to stdout if running in finspace
 write:{
-	if[logtodisk;@[neg logh;format x;()]];
-	if[logtomemory; `.usage.usage upsert x];
-	ext[x]}
+    if[.finspace.enabled;@[neg 1;format x;()]];
+    if[logtodisk;@[neg logh;format x;()]];
+    if[logtomemory; `.usage.usage upsert x];
+    ext[x]} 
 
 // extension function to extend the logging e.g. publish the log message
 ext:{[x]}
 
 // format the string to be written to the file
-format:{"|" sv -3!'x}
+format:$[`jsonlogs in key .proc.params;
```

Comment:
Do we have to do `jsonlogs if .finspace.enabled is true? Or is it still an option?

Reply:
It is still an option (command line), but maybe this should be an "OR" statement... Let me check

Reply:
No, definitely don't want an OR statement in there... Looks all good. There is still the option to set `jsonlogs to false if you want in finspace